### ISSUE TITLE
textwrap v0.15.1 is yanked.

### ISSUE THREAD

**Issue by charliie-dev** [rossmacarthur/sheldon#153]
I tried to install shelden via `cargo install shelden`, it gives me error:
```rust
    Updating crates.io index
  Installing sheldon v0.7.0
error: failed to compile `sheldon v0.7.0`, intermediate artifacts can be found at `/tmp/cargo-installThS3KI`

Caused by:
  failed to select a version for the requirement `textwrap = "^0.15.1"`
  candidate versions found which didn't match: 0.16.0, 0.15.0, 0.14.2, ...
  location searched: crates.io index
  required by package `clap v3.2.22`
      ... which satisfies dependency `clap = "^3.2.22"` of package `sheldon v0.7.0`
```

This repo requires `textwrap 0.15.1`, [code](https://github.com/rossmacarthur/sheldon/blob/5298ab09c6198de59f1464702e0c6ee1a70a3423/Cargo.lock#L909).

However it's had [been yanked 1 month ago](https://crates.io/crates/textwrap/versions), maybe it's time to upgrade the version?

**Comment by rossmacarthur** [comment]
Hmmm thats annoying, you should be able to install using `cargo install sheldon --locked`. We will have to upgrade to a later version of `clap` that doesn't depend on this version of `textwrap`

**Comment by charliie-dev** [comment]
I had downloaded the pre-build released instead, so at least things can still work.
Thanks for the reply!

**Comment by rossmacarthur** [comment]
Fixed in f2fb55d492786a9fa3917230be93fb3e691f4c26 which will be included in the next release
